src/st/np3/np3.h

```diff
@@ -121,7 +121,15 @@ extern Unkstruct_80180FE0 D_8018129C[];
 
 // *** Merman properties START ***
 
-extern u16 D_80181230[];     // pos TBL
+extern u16 D_80181230[]; // pos TBL
+extern u16 D_80181678[];
+extern u8 D_80181998[];
+extern u8 D_8018199C[];
+extern u8 D_801819A4[];
+extern u16 D_801819AC[];
+extern u8 D_801819C0[];
```

Comment:
```suggestion
extern u8 D_801819C0[12];
```
I know it should be `10`, but I am not sure how happy it is going to be alignment-wise

Reply:
I think with 10 it should be okay since it will get padded out to 32-bits. Seems to work at least for now.